themis.podspec

```diff
@@ -93,21 +93,23 @@ Pod::Spec.new do |s|
 
     # use `themis/themis-openssl` as separate target to use Themis with OpenSSL
     s.subspec 'themis-openssl' do |so|
-        # Enable bitcode for OpenSSL in a very specific way, but it works, thanks to @deszip 
+        # Enable bitcode for OpenSSL in a very specific way, but it works, thanks to @deszip
         so.ios.pod_target_xcconfig = {
             'OTHER_CFLAGS[config=Debug]'                => '$(inherited) -fembed-bitcode-marker',
             'OTHER_CFLAGS[config=Release]'              => '$(inherited) -fembed-bitcode',
             'BITCODE_GENERATION_MODE[config=Release]'   => 'bitcode',
             'BITCODE_GENERATION_MODE[config=Debug]'     => 'bitcode-marker'
         }
-        
-        # Xcode12, arm64 simulator issues https://stackoverflow.com/a/63955114
-        # disable building for arm64 simulator for now
-        
-        so.ios.pod_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
+
+        # As of version 1.0.2.20.1, GRKOpenSSLFramework binaries do not contain
+        # arm64 slices for iOS Simulator and macOS, and thus do not support
+        # Apple Silicon. Disable building Themis for Apple Silicon until
+        # GRKOpenSSLFramework gets proper arm64 support.
+        so.ios.pod_target_xcconfig  = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
         so.ios.user_target_xcconfig = { 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'arm64' }
-        
-        
+        so.osx.pod_target_xcconfig  = { 'EXCLUDED_ARCHS' => 'arm64' }
+        so.osx.user_target_xcconfig = { 'EXCLUDED_ARCHS' => 'arm64' }
```

Comment:
Have you tested that linting podspec works without these lines?

Reply:
It doesn't work without those. It's also not possible to build macOS apps for arm64 using GRKOpenSSL. That's the reason for arm64 exclusion.